tools/bqtag/setup.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import io
+import os
+import setuptools
+
+
+name = 'bqtag'
+description = 'Utility class for tagging BigQuery Tables and creating Authorized Views based on tags'
+version = '0.1.0'
+release_status = 'Development Status :: 3 - Alpha'
+dependencies = [
+    'google-cloud-bigquery >= 1.9.0',
+    'google-cloud-datacatalog >= 3.2.1'
+]
+extras = {}
+
+
+# Setup boilerplate below this line.
+package_root = os.path.abspath(os.path.dirname(__file__))
+
+readme_filename = os.path.join(package_root, 'README.md')
+with io.open(readme_filename, encoding='utf-8') as readme_file:
+    readme = readme_file.read()
+
+
+setuptools.setup(
+    name=name,
+    version=version,
+    description=description,
+    long_description=readme,
+    license='Apache 2.0',
+    classifiers=[
+        release_status,
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: Apache Software License',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
+        'Programming Language :: Python :: 3.8',
+        'Operating System :: OS Independent',
+        'Topic :: Internet',
+    ],
+    platforms='Posix; MacOS X; Windows',
+    packages=['bqtag'],
+    install_requires=dependencies,
+    extras_require=extras,
+    python_requires='>=3.6',
+    include_package_data=True,
+    zip_safe=False,
+)
```

Comment:
Blank line at the end of Python file (PEP8)

Reply:
Updated